lib/nerves_hub_link/socket.ex

```diff
@@ -481,7 +506,7 @@ defmodule NervesHubLink.Socket do
       _ = Client.handle_error(reason)
     end
 
-    rejoin(socket, topic, socket.assigns.params)
```

Comment:
why are the params removed?

Reply:
Sorry, I meant to comment. The slipstream default is to reuse the params that were sent in the initial join. With the addition of `features` channel, this was sending the generic firmware metadata params in the event of the features channel closing and rejoining.

I could have pattern matched on the channel name to selectively choose, or just let slipstream send the default original params (which are already static anyway, so no need to forcefully send)